Makefile

```diff
@@ -120,6 +119,27 @@ docker-push:
 	@echo ${DOCKERHUB_TOKEN} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
 	docker push ${IMG_W_TAG}
 
-help:
-	@grep -E '^[a-zA-Z_-]+:.*$$' $(MAKEFILE_LIST) | sort
+# Targets for release prep.
```

Comment:
targets' names convey this already. I think `# Targets to be run before submitting a new release` or something similar would be clearer

Reply:
sure. the child targets are self-explanatory. Will change the comment.